module/metrics/transaction.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/rs/zerolog"
 
+	"github.com/onflow/flow-go/engine/consensus/sealing/counters"
```

Comment:
cc @durkmurder 

Now I feel this module can be moved to a shared package, as its usage is quite common across the codebase. 

can be done in a separate PR